src/extension.ts

```diff
@@ -48,14 +55,21 @@ function getServerOptions(config: PhpactorConfig): ServerOptions {
     let serverOptions
     if (!config.remote.enabled) {
         // launch language server via stdio
-        serverOptions = {
+        serverOptions = <ServerOptions>{
             run: {
-                command: config.path,
-                args: ['language-server', ...config.launchServerArgs],
+                command: config.executablePath,
+                args: [config.path, 'language-server', ...config.launchServerArgs],
             },
             debug: {
```

Comment:
how is this used by vscode?

Reply:
It's a bit messy because it depends on the `composer` PR. Here I'm giving the user an option to optionally define the path to `php.exe`.

https://github.com/phpactor/vscode-phpactor/pull/61/commits/3b2781b3eb048bc28e397181d3464b842469b334#diff-04bba6a35cad1c794cbbe677678a51de13441b7a6ee8592b7b50be1f05c6f626R33

Both vscode-php-debug and vscode-php-intellisense have this. Altso the internal php validator has an option to define a path to PHP runtime.

Personally (although on Windows) I don't have php in my path, or have many different php versions that I have to switch between them.